src/components/repeat/ohri-repeat.component.tsx

```diff
@@ -210,11 +209,10 @@ export const OHRIRepeat: React.FC<OHRIFormFieldProps> = ({ question, onChange })
   encounterContext.sessionMode != 'view' &&
     nodes.push(
       <div>
-        {showAddButton(question.questionOptions.repeatOptions, counter) && (
+        {showAddButton(question.questionOptions.repeatOptions?.limit, counter) && (
           <Button
             renderIcon={() => <Add size={16} />}
-            className={styles.repeatButton}
-            kind="tertiary"
+            kind="ghost"
```

Comment:
@lucyjemutai do you need the above change? 

```suggestion
            kind="tertiary"
```

Reply:
no

Reply:
I just confirmed that we are using the "ghost" kind.